docs/ecctl-release-notes.asciidoc

```diff
@@ -12,3 +13,4 @@ include::release_notes/v1.0.0.adoc[]
 include::release_notes/v1.0.0-beta3.adoc[]
 include::release_notes/v1.0.0-beta2.adoc[]
 include::release_notes/v1.0.0-beta1.adoc[]
+include::release_notes/v1.1.0.adoc[]
```

Comment:
This line needs to be moved to the top of the list so it shows up properly on the docs page